src/components/Text.tsx

```diff
@@ -206,8 +208,10 @@ export function LinkText({
       className={linkText(bold, color, gradientFrom, gradientTo)}
       href={url}
       title={title}
-      rel="noopener noreferrer"
-      target="_blank"
+      {...(blank && {
```

Comment:
Let's remove `rel` attribute, because [all modern browsers include it automatically for `_blank`](https://stackoverflow.com/a/50709724/14385332) + let's rewrite this line:
```TS
target={targetBlank && '_blank'}
```
or if it won't work:
```TS
target={targetBlank ? '_blank' : '_self'}
```